### ISSUE TITLE
Check comments to date for info on incorrectly marked out not-titles

### ISSUE THREAD

**Issue by mialondon** [LibCrowds/libcrowds#658]
When checking results / designing the review interface we'll need to check for missing titles not marked as well as 'titles' incorrectly marked.

**Comment by alexandermendes** [comment]
Actually all of the "not a title" comments become somewhat redundant. If no one transcribes them then we won't get an annotation. But yep we can have comments displayed in our review interface.

**Comment by mialondon** [comment]
Some will probably get transcribed as it's easy to just get into the mode of typing the thing that appears in front of you (and depending on the zoom / screen size it mightn't always be obvious that it's a song or whatever).

**Comment by alexandermendes** [comment]
Sure yeah, definitely agreed that the comments should be visible via our review interface and there we should be able to delete/amend annotations as necessary.

**Comment by alexandermendes** [comment]
Yep, well the task really, the next task may well still be on the same page. For example, if we 3 tasks to transcribe 3 different marked up areas on a page, these will have to be rejected individually.

**Comment by mialondon** [comment]
I think that's an unexpected behaviour, people still might want to add tags or transcribe some text.

**Comment by alexandermendes** [comment]
Transcribing text would be kind of pointless as the purpose of rejecting the task is that the transcriptions won't be recorded! But OK, they might want to select tags. How should it work then? They select their option and from the dropdown and click OK, then...
